app/bot/openai.go

```diff
@@ -41,15 +45,33 @@ func NewOpenAI(authToken string, maxTokens int, prompt string, httpClient *http.
 	config.HTTPClient = httpClient
 
 	client := openai.NewClientWithConfig(config)
-	return &OpenAI{authToken: authToken, client: client, maxTokens: maxTokens, prompt: prompt,
-		nowFn: time.Now, superUser: superUser}
+	history := NewLimitedMessageHistory(5)
```

Comment:
5 is better to be a param and would be nice to pass it from main. I feel like we may want to play with this